### PR TITLE
Assignment4 @hgill @JessicaH46

### PR DESCRIPTION
**Pull request by dcox2005**
Front End
https://zealous-coast-08a1d1310.1.azurestaticapps.net/

API
https://wordleapiclassdavid.azurewebsites.net/swagger

### CODE REVIEW

File: wordle-web/pages/dailyGame.vue [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+  getDailyWordGame()
+  {
+    let currentDate = new Date().toJSON();
+    this.$axios.post('/DateWord/GetDailyWord', {
+      // date: new Date().toUTCString()
+      date: currentDate
+    })
+    .then((response) => {
+      this.word = response.data
+console.log(`WORD: ${this.word}`);
```
Comment (NazariyPogo) [inline wordle-web/pages/dailyGame.vue:132]: Make sure this doesn't stay in the end, unless you want it to.

**Review by NazariyPogo — COMMENTED** [review]
Everything looks pretty well done, I love the pretty theme that is used, instructions page is very informative, pages adapt well to size, daily game and stats are working. Just don't forget the ignores set in the PlayerServiceTests and to consider removing the daily word console log.

**Comment by hewrtarkhany** [comment]
i like the instruction page, it's well put together. over all the app is good

File: wordle-web/pages/instructionsPage.vue [hunk 2]
```diff
@@ -0,0 +1,90 @@
+<template>
+  <v-app style="background: #064a80">
```
Comment (mmwoodfo) [inline wordle-web/pages/instructionsPage.vue:2]: This is really nice. I gave you guys a couple of extra points for the additional effort on this page!

File: wordle-web/pages/dailyGame.vue [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+
+  getDailyWordGame() {
+    const currentDate = new Date().toJSON()
+    this.$axios
+      .post('/DateWord/GetDailyWord', {
+        date: currentDate,
+      })
+      .then((response) => {
+        this.word = response.data
+        console.log(`WORD: ${this.word}`) // left intentially to asses daily word is always same
```
Comment (mmwoodfo) [inline wordle-web/pages/dailyGame.vue:132]: Thank you - makes it easier. :)

File: wordle-web/pages/dailyGame.vue [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+import { GameState, WordleGame } from '~/scripts/wordleGame'
+import KeyBoard from '@/components/keyboard.vue'
+import GameBoard from '@/components/game-board.vue'
+import { Word } from '~/scripts/word'
+
+@Component({ components: { KeyBoard, GameBoard } })
```
Comment (mmwoodfo) [inline wordle-web/pages/dailyGame.vue:107]: Alot of the code in this typescript could probably be wrapped in a service and shared between the `dailyGame.vue` page and the `game.vue` page.

**Review by mmwoodfo — COMMENTED** [review]
Your daily word score doesn't seem to be saving and updating the `Last10DailyWords` page

**Comment by mmwoodfo** [comment]
@dcox2005 @hgill17  Did you guys do a PR review for Assignment 4?

**Comment by mmwoodfo** [comment]
@dcox2005 That's alright, if you'd like to do two for this final assignment I can count that additional PR as credit for this missed one. You may just need to remind me if you do end up doing two and don't see the points updated.

**Pull request closed without merging**